app/controllers/transcriptions_controller.rb

```diff
@@ -65,6 +68,9 @@ def export_group
     data_storage.zip_group_files(@transcriptions) do |zip_file|
       send_export_file zip_file
     end
+
+  rescue NoExportableTranscriptionsError => e
+    render_jsonapi_custom_not_found(e, use_sentry: false)
```

Comment:
If this is supposed to _not_ report to sentry, what's the difference between it and the default not_found error?

Reply:
The default not_found renderer doesn't include the exception message in the rendered response. I want the front-end to be able to access the message details for this error, which read "No exportable transcriptions found for group id ______"